### ISSUE TITLE
Pale images in pretrained model

### ISSUE THREAD

**Issue by muxgt** [brownvc/ganimorph#19]
Hi! First of all thanks for your work! I'm trying to reproduce your results with the pretrained model on Cats2Dogs. I'm using Google Colab and in TensorBoard the result looks like this:
![2](https://user-images.githubusercontent.com/54779582/64999109-4224c480-d8ef-11e9-828d-e424fd13baa6.jpg)
Since there is no "test" option I put steps_per_epoch = 0 and visualising after each epoch. Didn't change anything else in the code.
Do you know what the problem may be?

**Comment by Skylion007** [comment]
That is really, really bizarre. Could you send a link to the Colab?

**Comment by muxgt** [comment]
Yes, here it is. https://drive.google.com/open?id=1yRhvFm9rZgFzjBQIOL-7d2U7L3dc9yuv
You don't have .meta files in your pretrained models. Is that OK?
